src/CachePlugin.php

```diff
@@ -158,20 +158,20 @@ private function createCacheKey(RequestInterface $request)
      *
      * @param ResponseInterface $response
      *
-     * @return int|null
+     * @return int
      */
     private function getMaxAge(ResponseInterface $response)
     {
         if (!$this->config['respect_cache_headers']) {
-            return $this->config['default_ttl'];
+            return (int) $this->config['default_ttl'];
```

Comment:
Ok, I've updated the implementation. I'll probably turn off caching in my production app right now until this is resolved.